lib/ImageHelper.php

```diff
@@ -533,6 +533,9 @@ public static function analyze_url( $url ) {
 		$parts = PathHelper::pathinfo($tmp);
 		$result['subdir'] = ($parts['dirname'] === '/') ? '' : $parts['dirname'];
 		$result['filename'] = $parts['filename'];
+		// @todo remove this debug dump
```

Comment:
Do we still have to fix something here?

Reply:
I think that was only temporary while we were figuring some image stuff out. Removed and it's working fine.